scripts/ci/pin-to-cluster.sh

```diff
@@ -28,12 +28,12 @@ echo "::group::waiting until pinned"
         --host "/dnsaddr/ipfs-websites.collab.ipfscluster.io" \
         --basic-auth "${CLUSTER_USER}:${CLUSTER_PASSWORD}" \
         status "$PIN_CID" | tee cluster-pin-status
-    if [[ $(jq '.peer_map[].status' cluster-pin-status | grep '"pinned"' | wc -l) -ge 2 ]]; then
-        echo "Got 2 pin confirmations, finishing the workflow"
```

Comment:
:thought_balloon: Reduced the number of confirmations to 1 to decouple release process from the collab cluster state until we have internal refactor from https://github.com/ipshipyard/waterworks-infra/issues/177 finished.